PhysicsTools/PatAlgos/python/recoLayer0/electronPFIsolationDepositsPAT_cff.py

```diff
@@ -1,14 +1,14 @@
 import FWCore.ParameterSet.Config as cms
 
-from CommonTools.ParticleFlow.Isolation.electronPFIsolationDepositsPFBRECO_cff import *
+import CommonTools.ParticleFlow.Isolation.electronPFIsolationDepositsPFBRECO_cff as m
```

Comment:
should this better be ```as _m``` so that someplace else with ```electronPFIsolationDepositsPAT_cff import *``` doesn't end up with a rogue ```m```?